script launch error

@claude

ok https://mana-meeples-web.onrender.com/ is loading the index.html page which is great and I can see the html in inspect but I'm getting the following errors:

(index):12   GET https://mana-meeples-web.onrender.com/shop/assets/index-DDaCbxWF.js net::ERR_ABORTED 404 (Not Found)
/manifest.json:1   GET https://mana-meeples-web.onrender.com/manifest.json net::ERR_QUIC_PROTOCOL_ERROR.QUIC_NETWORK_IDLE_TIMEOUT

I can see in main.tsx there is an issue with this line: import App from "./App";, i've just changed it to /app and that seems to work on that file

but then on render deploy api:

2025-10-19T01:41:50.810816872Z ==> Downloading cache...
2025-10-19T01:41:50.812896657Z ==> Cloning from https://github.com/muppetbrown/mana_meeples_singles_market
2025-10-19T01:41:51.342463442Z ==> Checking out commit 4831bfcd06258c200765c9a0ba0e28c51215b2f7 in branch main
2025-10-19T01:42:05.179732581Z ==> Transferred 116MB in 8s. Extraction took 5s.
2025-10-19T01:42:09.273025832Z ==> Installing dependencies with pnpm...
2025-10-19T01:42:09.357480041Z ==> Requesting Node.js version 20.x
2025-10-19T01:42:09.62724276Z ==> Using Node.js version 20.19.5 via /opt/render/project/src/package.json
2025-10-19T01:42:09.652447803Z ==> Docs on specifying a Node.js version: https://render.com/docs/node-version
2025-10-19T01:42:10.258969671Z Scope: all 3 workspace projects
2025-10-19T01:42:10.999673618Z 
2025-10-19T01:42:11.021984973Z Done in 1.2s
2025-10-19T01:42:11.072314643Z ==> Running build command 'corepack enable && pnpm install --no-frozen-lockfile && pnpm --filter ./apps/web run build'...
2025-10-19T01:42:11.604046327Z Scope: all 3 workspace projects
2025-10-19T01:42:12.379584901Z 
2025-10-19T01:42:12.398705109Z Done in 1.1s
2025-10-19T01:42:12.872211633Z 
2025-10-19T01:42:12.872245095Z > web@0.1.0 build /opt/render/project/src/apps/web
2025-10-19T01:42:12.872249165Z > vite build
2025-10-19T01:42:12.872252146Z 
2025-10-19T01:42:13.071933837Z vite v5.4.20 building for production...
2025-10-19T01:42:13.354029795Z transforming...
2025-10-19T01:42:13.378244513Z ✓ 3 modules transformed.
2025-10-19T01:42:13.380163728Z x Build failed in 283ms
2025-10-19T01:42:13.380218091Z error during build:
2025-10-19T01:42:13.380228272Z Could not resolve "./app" from "src/main.tsx"
2025-10-19T01:42:13.380231422Z file: /opt/render/project/src/apps/web/src/main.tsx
2025-10-19T01:42:13.380234282Z     at getRollupError (file:///opt/render/project/src/node_modules/.pnpm/rollup@4.52.5/node_modules/rollup/dist/es/shared/parseAst.js:401:41)
2025-10-19T01:42:13.380236442Z     at error (file:///opt/render/project/src/node_modules/.pnpm/rollup@4.52.5/node_modules/rollup/dist/es/shared/parseAst.js:397:42)
2025-10-19T01:42:13.380239273Z     at ModuleLoader.handleInvalidResolvedId (file:///opt/render/project/src/node_modules/.pnpm/rollup@4.52.5/node_modules/rollup/dist/es/shared/node-entry.js:21565:24)
2025-10-19T01:42:13.380242243Z     at file:///opt/render/project/src/node_modules/.pnpm/rollup@4.52.5/node_modules/rollup/dist/es/shared/node-entry.js:21525:26
2025-10-19T01:42:13.392944516Z /opt/render/project/src/apps/web:
2025-10-19T01:42:13.392965747Z  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  web@0.1.0 build: `vite build`
2025-10-19T01:42:13.392967977Z Exit status 1
2025-10-19T01:42:13.427385696Z ==> Build failed 😞
2025-10-19T01:42:13.427413197Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys
2025-10-19T01:42:13.495031736Z ==> Requesting Node.js version 20.x
2025-10-19T01:42:13.62942642Z ==> Using Node.js version 20.19.5 via /opt/render/project/src/package.json
2025-10-19T01:42:13.65459175Z ==> Docs on specifying a Node.js version: https://render.com/docs/node-version

--- a/apps/web/src/main.tsx
+++ b/apps/web/src/main.tsx
@@ -1,6 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import App from "./app";
+import App from "./App";
 
 ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>

